### ISSUE TITLE
Could you please update the web ui ?

### ISSUE THREAD

**Issue by zxl777** [willpowell8/LocalizationKit_iOS#29]
- New language will copy the strings from default language strings.
- Delete the selected key

**Comment by willpowell8** [comment]
Yes I am just trying to get the last few UI fixes in to resolve issues. Then we will be good to go with both of these new updates.
